packages/expo-module-scripts/babel.config.node.js

```diff
@@ -0,0 +1,27 @@
+module.exports = () => ({
+  presets: [
+    [
+      require('@babel/preset-env'),
+      {
+        targets: {
+          node: '12.0.0',
+        },
+        modules: false,
+      },
+    ],
+    require('@babel/preset-typescript'),
+  ],
+  plugins: [
+    require('babel-plugin-dynamic-import-node'),
+    require('@babel/plugin-proposal-export-namespace-from'),
+    require('@babel/plugin-proposal-class-properties'),
+    [
+      require('@babel/plugin-transform-modules-commonjs'),
+      {
+        lazy: /* istanbul ignore next */ (source) => true,
+      },
+    ],
+    require('@babel/plugin-proposal-optional-chaining'),
+    require('@babel/plugin-proposal-nullish-coalescing-operator'),
+  ],
```

Comment:
```suggestion
  plugins: [
    require('@babel/plugin-proposal-export-namespace-from'),
    [
      require('@babel/plugin-transform-modules-commonjs'),
      {
        lazy: /* istanbul ignore next */ (source) => true,
      },
    ],
  ],
```

Reply:
Just dropped the safe-to-drop plugins, while keeping the plugins that might affect code.

Reply:
Thanks for testing. That's too bad about babel-plugin-dynamic-import-node but makes sense that since we support non-compliant import specifiers (`import './a'`) that people would write `import('./a')` too.